src/Shell/Task/ModelTask.php

```diff
@@ -299,6 +300,10 @@ public function findTableReferencedBy($schema, $keyField)
                 if (!isset($constraintInfo['references'])) {
                     continue;
                 }
+                $length = strlen($this->tablePrefix);
```

Comment:
It's a valid point when you have to use Cake on a legacy DB with for example German table names.